<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="update_button">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>განახლება</string>
    </property>
   </widget>
   <widget class="QLabel" name="cash">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>474</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cash_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>474</y>
      <width>52</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ნაღდი:</string>
    </property>
   </widget>
   <widget class="QLabel" name="card_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>498</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>უნაღდო</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="all_payments">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>897</width>
      <height>440</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1001</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>სახელი</string>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>გვარი</string>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ტელეფონი</string>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>მომსახურება</string>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>გადახდის მეთოდი</string>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>თარიღი</string>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>თანხა / წუთი</string>
     </property>
     <property name="font">
      <font>
       <family>BPG Nino Mtavruli</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="pay_by_minutes_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>522</y>
      <width>238</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>გადახდა აბონემენტის წუთებით:</string>
    </property>
   </widget>
   <widget class="QLabel" name="card">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>498</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="total_daily">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pay_by_minutes">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>522</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="total_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>264</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BPG Nino Mtavruli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>მიმდინარე დღის შემოსავალი სულ: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
